src/Donations/PaymentMethods/NewPaypal.tsx

```diff
@@ -77,17 +77,7 @@ function NewPaypal({
     payDonationFunction("paypal", "paypal", data);
   };
 
-  const onCancel = (data, actions) => {
```

Comment:
Do we really do not want to show a [cancel message](https://developer.paypal.com/docs/checkout/standard/customize/show-cancellation-page/) for the user who intentionally aborted the Paypal donation?

Reply:
We could design a proper cancellation page instead of reusing the error page for the cancellation message?

Reply:
For comparison, I took a look at how the official [PayPal plugin for WooCommerce](https://wordpress.org/plugins/woocommerce-paypal-payments/) (developed by WooCommerce & PayPal) handles the case. There, too, nothing happens when you close the PayPal window.
You can check it out at the [Plant-for-the-Planet shop](https://www.thegoodshop.org/), for example.